1533 Google Drive Delete File Action

MayankMadan97 opened the pull request:
Added Delete file aciton to Google Drive components.
Fixes hhttps://github.com/bytechefhq/bytechef/issues/1533

**Type of change**
New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**
Junit
Test/TestIntegration
Manually test with personal Google Drive creds

## Checklist:

- [✔ ] My code follows the style guidelines of this project
- [ ✔] I have performed a self-review of my own code
- [ ✔] I have commented my code, particularly in hard-to-understand areas
- [ ✔] I have made corresponding changes to the documentation
- [ ✔] My changes generate no new warnings
- [ ✔] I have added tests that prove my fix is effective or that my feature works
- [ ✔] New and existing unit tests pass locally with my changes

monikakuster commented on server/libs/modules/components/google/google-drive/src/main/java/com/bytechef/component/google/drive/action/GoogleDriveDeleteFileAction.java:
```diff
@@ -0,0 +1,61 @@
+                            .description("The id of a file to delete.")
+                            .options((ActionOptionsFunction<String>) GoogleDriveUtils::getFileOptions)
+                            .required(true))
+            .output()
```
Since the output is empty `output()` can be removed.

MayankMadan97 replied:
I've updated the return type to Object instead of void, since with void you can't chain the perform method call to the action. It needs to return something for that. However, I removed output() from the chain, as there's generally no need for an output when performing a deletion operation.

monikakuster commented on server/libs/modules/components/google/google-drive/src/main/java/com/bytechef/component/google/drive/action/GoogleDriveDeleteFileAction.java:
```diff
@@ -0,0 +1,61 @@
+    private GoogleDriveDeleteFileAction() {
+    }
+
+    public static Void perform(
```
Use `void` (primitive type) as return type

MayankMadan97 replied:
I've updated the return type to Object instead of void, since with void you can't chain the perform method call to the action. It needs to return something for that. However, I removed output() from the chain, as there's generally no need for an output when performing a deletion operation.

monikakuster commented:
@MayankMadan97,You have missed steps 4 and 5 from the [guide](https://github.com/bytechefhq/bytechef/blob/master/CONTRIBUTING.md#server-side), which are crucial for ensuring that your changes align with our project's standards and processes.

Additionally, it's a good practice to include the issue number at the beginning of your commit message, followed by a concise description of the changes made. This helps in tracking the progress of specific issues and provides context for the commit, making it easier for others to understand the purpose and scope of your changes.

For example: `1533 - added delete file action`.

MayankMadan97 commented:
> @MayankMadan97, you have missed steps 4 and 5 from the [guide](https://github.com/bytechefhq/bytechef/blob/master/CONTRIBUTING.md#server-side), which are crucial for ensuring that your changes align with our project's standards and processes.
> 
> Additionally, it's a good practice to include the issue number at the beginning of your commit message, followed by a concise description of the changes made. This helps in tracking the progress of specific issues and provides context for the commit, making it easier for others to understand the purpose and scope of your changes.
> 
> For example: `1533 - added delete file action`.

Sorry for the oversight! I've made the changes you requested, and I also generated the .json file in the component's resources directory. I ran all the necessary checks on my machine as per the guidelines  before pushing the updates.

I've updated the return type to Object instead of void, since with void you can't chain the perform method call to the action. It needs to return something for that. However, I removed output() from the chain, as there's generally no need for an output when performing a deletion operation.
